### PR TITLE
Workaround typing_extensions module requirement

### PR DESCRIPTION
**Pull request by jwhonce**
This requirement was missed.

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

**Comment by jwhonce** [comment]
@lsm5 FYI, keeping this as draft until we know outcome of packaging needs.

File: requirements.txt [hunk 1]
```diff
@@ -4,4 +4,5 @@ urllib3>=1.24.2
 pyxdg>=0.26
 sphinx
 wheel
+typing_extensions; python_version < '3.8'
```
Comment (lsm5) [inline requirements.txt:7]: @jwhonce help me read this line, does this say: `Explicitly require typing_extensions for any python less than 3.8?`

If that is the case, we still need to include typing-extensions in rhel8.

**Comment by lsm5** [comment]
@jwhonce remind me what exactly I need to test to greenlight this for 8.5?

**Comment by jwhonce** [comment]
@lsm5 Do `make` and `make tests` would for your environment?

**Comment by lsm5** [comment]
RE: rhel8.5, there's `python3-pytoml` and `python39-toml`. python3-pytoml would be the recommended version to use because it would be supported for the entire lifetime of rhel8. How bad is it if we depend on python3-pytoml in the rpm packaging? 


btw, rpm builds fine for now on rhel8.5 

RE: `make tests`, I see: `DEBUG:podman.ssh_adapter:Waiting on /run/user/1000/podman/podman-forward-a97c89ca551bad70c129.sock` , even after installing podman 3.2 and enabling podman.sock. Let me know if you prefer a tmate session.

**Comment by jwhonce** [comment]
@lsm5 `DEBUG:podman.ssh_adapter:Waiting on...` only occurs when running with debugging on and is telling you the client is waiting on the external ssh client. This is not an error. Should the ssh client never forward the connection, you will then receive a TimeoutExpired exception/message. That is an error.

**Comment by lsm5** [comment]
> @lsm5 `DEBUG:podman.ssh_adapter:Waiting on...` only occurs when running with debugging on and is telling you the client is waiting on the external ssh client. This is not an error. Should the ssh client never forward the connection, you will then receive a TimeoutExpired exception/message. That is an error.

remind me, is there anything else for me to do on this part then? 


> If you use different spec files or differentiate between RHEL8 and RHEL9 in one spec file that is fine with me. Do you need me to tweak requirements.txt for which toml package to depend on ?

I think that's not needed right now, but if brew complains when we do the actual builds, then maybe.

**Comment by lsm5** [comment]
LGTM as far as build is concerned. I haven't had the chance to follow up on testing though.

**Comment by jwhonce** [comment]
@mwhahaha PTAL Thanks.

**Pull request merged**